bigchaindb/models.py

```diff
@@ -0,0 +1,209 @@
+from functools import reduce
```

Comment:
Overall not a big fan of this module, since I think a lot of it could be turned into just validation functions or block util functions, but ¯\_(ツ)_/¯ tbtf.